controllers/serverController.js

```diff
@@ -286,15 +297,46 @@ module.exports = {
     },
 
     rest: {
+        all: function (req, res) {
```

Comment:
the only difference between this and the regular all (which is also a regular json route) is that there is no check if a user  is logged in, do you need this?